Odata-docs/webapi-8/fundamentals/client-driven-paging.md

```diff
@@ -0,0 +1,173 @@
+---
+title:  "Client-driven paging in ASP.NET Core OData 8"
+description: Client-driven paging in ASP.NET Core OData 8.
+date:   2023-01-10
+ms.date: 1/10/2023
+author: habbes
+ms.author: clhabins
+---
+
+# Client-driven paging in ASP.NET Core OData 8
+**Applies To**:[!INCLUDE[appliesto-webapi](../../includes/appliesto-webapi-v8.md)]
+
+The client can implement paging by telling the OData service how many items to return for a given request. The `$top` query option is used to limit the number of results and the `skip` query option used to specify the offset. For example `GET /Customers?$skip=3$top=5` will return up to 5 items starting from the 4th item in the collection (i.e. after skipping the first 3).
```

Comment:
The words "_client can implement paging_" do not do it for me... Especially the word "implement". Maybe "_The client can request the OData service to return a specific number of results_"

Reply:
What I meant by that is the fact that on the client-side you can have paging UI where the user can select a specific page number, or next/prev pages, and the logic that manages this, to keep track of pages, etc. lies on the client side.

Reply:
I rephrased it based on the suggestion.